scripts/utils/call.js

```diff
@@ -165,6 +165,19 @@ async function getState(privateKey, contract, token) {
   return state
 }
 
+async function getZRC2State(wallet, token) {
```

Comment:
This implementation is correct but you are getting the whole contract state only to get balance. Can we implement this code like below which is only getting balance?

async function getZRC2State(wallet, token) {
  const wallet = String(this.wallet).toLowerCase();
  const tokenContract = zilliqa.contracts.at(token);
  const { result } await tokenContract.getSmartContractSubState(
    contract,
    'balances',
    [wallet]
  );
  if (result && result[field] && result[field][wallet]) {
    return result[field][wallet];
  } 

  return '0';
}

Reply:
@ashishnair-zil Thank you, will use this.

Reply:
@razill Will update this in upcoming versions. Thank you.